timelock-impl/src/main/java/com/palantir/atlasdb/timelock/adjudicate/FeedbackHandler.java

```diff
@@ -149,7 +148,13 @@ HealthStatus pointFeedbackHealthStatus(ConjureTimeLockClientFeedback healthRepor
     private HealthStatus getHealthStatusForService(EndpointStatistics endpointStatistics,
             double rateThreshold,
             long p99Limit,
-            double errorRateProportion) {
+            double errorRateProportion,
+            double p99Multiplier) {
+
+        // Outliers indicate badness, regardless of request rate
+        if (endpointStatistics.getP99() > p99Limit * p99Multiplier) {
```

Comment:
On one hand I'd prefer not to add complexity - on the other I think we should special case a flat 0 for the request rate, because this is backed by an exponential metric registry and so an old, slow request might end up hanging for more than we would like.